### DISCUSSION TITLE
Price List / Price Selection Strategy / Discounts

### DISCUSSION THREAD

**Discussion by juanzgc** [medusajs/medusa#3683] [Q&A]
## Customizability
Currently, `Price Lists` don't enable high customizability especially when using Sales Channels (not supported).

What if we can propose higher functionality for `Price Lists`, similar to `Discounts`?
Following the current Discounts model, we could achieve the addition of Adding Rules and Conditions that if met, could update the pricing of a product dynamically.

![Discounts](https://user-images.githubusercontent.com/24414523/229379311-42bdd40f-36e7-4351-b338-2945aea8650f.png)

This could enable updating the price of a product on Weekends, Holidays, etc... Without having to create an individual Price List for each start/end date.

Admins would then be able to customize the Price List, using a wide range of options such as the Order date, time, number of available stock, etc.

A similar model used by [Airbnb that enables hosts to customize their pricing](https://www.airbnb.com/help/article/2061):
<img width="500" alt="image" src="https://user-images.githubusercontent.com/24414523/229379553-0e4bf396-5179-4ab5-af3c-c34c2e819faf.png">

## Sales Channel

Now, what if we could also customize by the Sales Channel? An Admin may want to enable a `Discount` for a certain Sales Channel, but **not** all Sales Channels.

Currently both `Price Lists` and `Discounts` support the `region` condition. But I think we should also enable `Price Lists` and `Discounts` to support **Sales Channels**.

## A mix of both?

Transforming the `Price List` Model into a Rule/Condition based Model could potentially improve the Admin experience and enable higher customizations. Adding the `Sales Channel` condition, would improve the customization even greater because now you'd be able to add custom pricing based on Stock Location.

**Comment by danvayn** [comment]
I support this because it fits into the idea of a variable pricing model (name-your-price, for example) which I think should be supported by default.
